### PR TITLE
Add postgresql repository

### PR DESCRIPTION
**Pull request by carlosdauden**
Add postgresql repository to get the latest version of postgresql-client

### CODE REVIEW

**Comment by pedrobaeza** [comment]
I think the idea is to base this on Alpine as soon as we get rid of the nasty bug we have, so I'm not sure this is the best option for long term.

**Comment by blaggacao** [comment]
For alpine it goes arround this

```
RUN echo "@edge http://dl-cdn.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories \
    && apk --no-cache add "postgresql@edge<9.7" "postgresql-contrib@edge<9.7"
```

**Comment by lasley** [comment]
Update on the stack overflow - I'm still able to replicate it using our email templates, even with our patch. We may be a ways off from Alpine 😦 . At this point, I'm 99% that lxml is the culprit - it's the only lib involved in everything that fails. But then again, Odoo uses lxml for basically everything, so it makes sense it's in all the stacks.

File: Dockerfile [hunk 1]
```diff
@@ -61,6 +61,8 @@ ENV OPENERP_SERVER=/opt/odoo/auto/odoo.conf \
 
 # Other requirements and recommendations to run Odoo
 # See https://github.com/$ODOO_SOURCE/blob/$ODOO_VERSION/debian/control
+RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ xenial-pgdg main' >> /etc/apt/sources.list.d/postgresql.list
```
Comment (Yajo) [inline Dockerfile]: You are adding the ubuntu version of the repos, please use Debian's if available.

**Comment by Yajo** [comment]
Hi all, I added #16 to track all Alpine stuff, so we have some meeting point about it and have no more need to keep on talking about it on every issue.

About the PR itself, just one requirement.

BTW, since we still did not fix #6, did you try building this Dockerfile locally and did it work, @carlosdauden?

**Pull request merged**